scripts/2015_nips_paper/run/run_commands.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# Run all commands in commands.txt. Modify this to run on a cluster.
+cat "../commands.txt" | while read line; do eval "$line"; done
```

Comment:
This would only work sequentially because only then it is guaranteed that the ensemble creation for an ensemble of size 50 is done after fitting the model. If all commands were started in parallel, this would not work I guess. If that's the case, could you please make sure that the two different command types are written in two separate files and update the readme?

Reply:
I see, could you please add a comment about this to `create_commands.sh` and `run_commands.sh`?

Reply:
Yep, I will do that